includes/editor-templates/global.php

```diff
@@ -35,6 +35,27 @@
 			<# } #>
 			<div class="elementor-add-section-drag-title"><?php echo esc_html__( 'Drag widget here', 'elementor' ); ?></div>
 		</div>
+		<div class="e-view e-con-flex-container e-con-select-type">
```

Comment:
Why do we need this class `e-con-flex-container`? It references `flex` specifically.